src/app/containers/App.js

```diff
@@ -21,15 +21,24 @@ const monitorPosition = location.hash;
 let monitor;
 let selectedTemplate;
 let testTemplates;
-chrome.storage.local.get({
-  ['monitor' + monitorPosition]: 'InspectorMonitor',
-  'test-templates': null,
-  'test-templates-sel': null
-}, options => {
-  monitor = options['monitor' + monitorPosition];
-  selectedTemplate = options['test-templates-sel'];
-  testTemplates = options['test-templates'];
-});
+
+// Electron: Not supported some chrome.* API
+if (chrome.storage.local) {
+  chrome.storage.local.get({
+    ['monitor' + monitorPosition]: 'InspectorMonitor',
+    'test-templates': null,
+    'test-templates-sel': null
+  }, options => {
+    monitor = options['monitor' + monitorPosition];
+    selectedTemplate = options['test-templates-sel'];
+    testTemplates = options['test-templates'];
+  });
+} else {
+  monitor = localStorage.getItem('monitor' + monitorPosition]) || 'InspectorMonitor';
+  selectedTemplate = localStorage.getItem('test-templates-sel');
+  testTemplates = localStorage.getItem('test-templates');
```

Comment:
```js
selectedTemplate = localStorage.getItem('test-templates-sel');
testTemplates = localStorage.getItem('test-templates');
```

These 2 lines can be removed at all as we already have this in [remotedev-app](https://github.com/zalmoxisus/remotedev-app/blob/master/src/app/components/TestGenerator.js#L23-L24) in case there're undefined.